CHANGELOG.md

```diff
@@ -1,8 +1,16 @@
 # Unreleased
 
 ### Changes
+
 * [IMPROVEMENT] Add mobile vitals frequency configuration. See [#876][]
 * [IMPROVEMENT] Include the exact model information in RUM `device.model`. See [#888][]
+* [FEATURE] Allow filtering outgoing logs with a status threshold. See [#867][]
+
```

Comment:
This change might look weird, but it's fixing the problem we have in `CHANGELOG` on `develop`, see [1.11.0 / 13-06-2022](https://github.com/DataDog/dd-sdk-ios/blob/develop/CHANGELOG.md#1110--13-06-2022) record.

Reply:
It's missing the links to `#867` and `#888` at the end of the file, but it might be best to fix it when we merge back `feature/v2` to `develop`

Reply:
Yes 👍, or when we format the `CHANGELOG.md` in next release (I use [pimpmychangelog](https://github.com/pcreux/pimpmychangelog) for this).